aeternity/transactions.go

```diff
@@ -38,7 +38,7 @@ func createSignedTransaction(txRaw []byte, signatures [][]byte) (rlpRawMsg []byt
 
 // SpendTx create a spend transaction
 // see https://github.com/aeternity/protocol/blob/epoch-v0.22.0/serializations.md
-func SpendTx(senderID, recipientID, payload string, amount, fee int64, ttl, nonce uint64) (rlpRawMsg []byte, err error) {
+func SpendTx(senderID, recipientID string, amount, fee, ttl, nonce uint64, payload string) (rlpRawMsg []byte, err error) {
```

Comment:
Shuldn't this use also big.Int (or BigInt) ? the max value for uint64 is ~1.8e19

Reply:
Strict bigint support requires more changes upstream, like the CLI. The idea was to first make everything that won't be negative uint64, and then afterwards be strict on BigInt. This saves time, especially since BigInt isn't properly printed by terminal.go yet.